lib/modules/platform/gitlab/readme.md

```diff
@@ -2,39 +2,46 @@
 
 ## Authentication
```

Comment:
I found this issue:

- #21121

Should this PR automatically close the issue above?